src/renderer/marketplace/exchanges/exchanges-details.jsx

```diff
@@ -319,7 +322,7 @@ class ExchangesDetailsComponent extends PureComponent {
 		}
 	};
 
-	renderAffiliateLinkButton = url => (
+	renderLinkButton = ({ url, isAffiliate = false, text = 'SIGN UP' }) => (
```

Comment:
In general it is better to do it as a component instead of a render method

Reply:
Improved on 8d9286b